cmd/bm-client/handlers/create_account.go

```diff
@@ -28,14 +27,31 @@ func CreateAccount(vault *vault.Vault, bmAddr, name, organisation, routing, toke
 
 	fmt.Printf("* Checking if address is already known in the resolver service: ")
 	ks := container.GetResolveService()
-	_, err = ks.Resolve(addr.Hash())
+	_, err = ks.ResolveAddress(addr.Hash())
 	if err == nil {
 		fmt.Printf("\n  X it seems that this address is already in use. Please specify another address.")
 		fmt.Println("")
 		os.Exit(1)
 	}
 	fmt.Printf("not found. This is a good thing.\n")
 
+	// Check token
+	fmt.Printf("* Checking token format and extracting data: ")
+	it, err := invite.ParseInviteToken(token)
+	if err != nil {
+		fmt.Printf("\n  X it seems that this token is invalid")
+		fmt.Println("")
+		os.Exit(1)
+	}
```

Comment:
Token should be verified with token.Verify function
https://github.com/bitmaelum/bitmaelum-suite/blob/a36d8a57674a5ab26bcf68d85c123b8ad7580ac0/internal/invite/signature.go#L80

Reply:
No need for checking. We only extract address and routing id from the token. Checks are done on the mailserver when pushing the account to it.